Scripts/Editor/LTCGI_ControllerBake.cs

```diff
@@ -152,10 +152,10 @@ internal void BakeLightmap(bool bakery, bool followup)
             EditorUtility.DisplayProgressBar("Preparing LTCGI bake", "Making LTCGI_Screens emissive", 0.5f);
 
             // make screen emissive
-            var allScreens = GameObject.FindObjectsOfType<LTCGI_Screen>();
+            var allScreens = gameObject.scene.GetRootGameObjects().SelectMany(go => go.GetComponentsInChildren<LTCGI_Screen>(true));
             foreach (var scr in allScreens)
             {
-                if (scr.LightmapChannel == 0) continue;
+                if (scr.LightmapChannel == 0 || (!scr.gameObject.activeSelf && (!scr.TryGetComponent(out LTCGI_BakeReset bakeReset) || !bakeReset.Reenable))) continue;
```

Comment:
Why this line? This means it won't bake disabled GameObjects. Judging by #14 the interest is rather in the opposite direction.

Reply:
If that is all there is to it, I may just merge this and the other one and then add a toggle for "bake when disabled".

Reply:
I thought it would make sense when you don't want to bake some objects by disabling it.

Reply:
It would make sense to disable baking when the gameobject is disabled, but not the mesh renderer. I think I was having this issue with usharp video player where I didn't want the light mesh to show or it'd be in the way of the actual video screen.